stl/inc/chrono

```diff
@@ -5566,18 +5655,39 @@ struct _Chrono_formatter {
             _Fmt_align::_Left, [&](auto _Out) { return _Fmt_write(_STD move(_Out), _Stream.view()); });
     }
 
+    _NODISCARD bool _Needs_localization(const _Chrono_specs<_CharT>& _Spec) noexcept {
+        if (_Spec._Modifier != '\0') {
+            return true;
+        }
+
+        // [tab:time.format.spec]
+        switch (_Spec._Type) {
+        case 'a':
+        case 'A':
+        case 'b':
+        case 'B':
+        case 'h':
+        case 'z':
+        case 'Z':

```

Comment:
Do you want to just populate this with all of the type specifiers that will end up here?  Or were you planning to add to this as we added `formatter`s/`operator<<`s for each type?  I think we'll also need to include `%c`, `%x`, `%X` looking at Table 101.

Reply:
Hmm, I think we need to rename this function. It was conceived as a way to figure out specifiers can throw, so we can explicit about `ok()` checking. `%c` uses locale, but I don't think it can ever be out of bounds. Maybe `_Type_needs_bounds_checking`?

Reply:
I _think_ `%c` will actually require more extensive bounds-checking because `put_time` [says that `%c` uses all fields of `tm` so there is even more opportunity for out-of-bounds issues?

Reply:
I want to say that `%c` is used for a point in time, measured from some base. You can't have an invalid seconds, because you derive the seconds from the amount of time elapsed.

Reply:
I'm trying to work out what type would be used to format `%c` and I think it would probably be `time_point`s right?  (e.g., `sys_time`, etc.).  If we would just use `time_point::time_since_epoch()` then I think you're right.